src/js/app.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function () {
+  // Hamburger Menu & Anchor Jump
+  // This exists on every page.
+
+  $('.hamburger').click(function () {
+    $(this).toggleClass('is-active')
+    $('.hamburger-active-menu').fadeToggle()
+  })
+
+  $('a[href^="#"]').anchorjump()
+
+  var url = window.location.href
+
+  $("meta[property='og:url']").attr('href', url)
+  $('link[rel=canonical]').attr('content', url)
```

Comment:
This doesn't work.  The point of canonical is to merge identical links.  We don't want to use what's in the browser b/c it renders the following as the same thing:

 - http://example.com/foo
 - http://example.com/foo/
 - http://example.com/foo/index.html

The canonical url should be http://example.com/foo

Reply:
@theycallmeswift 
would this work better?

```
  var location = window.location
  var pathname = location.pathname.replace('index.html', '')
  var canonicalUrl = location.protocol + '//' + location.host + pathname
```